lib/plist.c

```diff
@@ -138,6 +136,8 @@ xbps_array_foreach_cb_multi(struct xbps_handle *xhp,
 		}
 	}
 
+	reserved = slicecount * maxthreads;
```

Comment:
make sure reserved isn't initialized beforehand. This way we get a compile time warning when it's used before it's set to the correct value

Reply:
Done, not sure if it actually gives a warning, because the pointer is passed arround.